internal/config/cgmgr/cgmgr.go

```diff
@@ -144,6 +144,12 @@ func VerifyMemoryIsEnough(memoryLimit int64) error {
 // createSandboxCgroup takes the sandbox parent, and sandbox ID.
 // It creates a new cgroup for that sandbox, which is useful when spoofing an infra container.
 func createSandboxCgroup(sbParent, containerID string, mgr CgroupManager) error {
+	// If we are running systemd as cgroup driver then we would rely on
+	// systemd to create cgroup for us
+	if mgr.IsSystemd() {
+		return nil
+	}
+
```

Comment:
what would you think about returning in `systemd.go:CreateSandboxCgroup` and dropping this check? we already have a spot where we know the systemd cgroup driver is being used. We could even compress this function into the cgroupfs function as well.

Reply:
@haircommander Makes sense.

About, raising PR to master and back merging to prior releases - upto version `1.25` we use `containers/podman/cgroups.go:New(path string, resources *spec.LinuxResources)` for creating `cgroupfs` cgroups, while from `1.26` we use `containers/common:cgroups_linux.go:New(path string, resources *configs.Resources)` so I'll have to raise two separate PRs IMHO.

Reply:
yeah you may have to open multiple, but I think it will make the branches more consistent. If you'd rather not have the paperwork, one of the maintainers can do it, though we'd certainly appreciate your doing it if you feel like it!